tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+    "compilerOptions": {
+        "target": "es2022",
+        "moduleResolution": "node16",
+
+        "module": "nodenext",
+        "resolveJsonModule": false,
```

Comment:
will be overwritten to `true` (in line 18)?

Reply:
ehehe, true
```suggestion
```